checker/raw_result.go

```diff
@@ -84,3 +54,60 @@ type Tool struct {
 	Desc          string
 	ConfigFiles   []File
 }
+
+// SecurityPolicyData contains the raw results
+// for the Security-Policy check.
+type SecurityPolicyData struct {
+	// Files contains a list of files.
+	Files []File
+}
+
+// BinaryArtifactData contains the raw results
+// for the Binary-Artifact check.
+type BinaryArtifactData struct {
+	// Files contains a list of files.
+	Files []File
+}
+
+// DependencyUpdateToolData contains the raw results
+// for the Dependency-Update-Tool check.
+type DependencyUpdateToolData struct {
+	// Tools contains a list of tools.
+	// Note: we only populate one entry at most.
+	Tools []Tool
+}
+
+// BranchProtectionData contains the raw results
+// for one branch.
+//nolint:govet
+type BranchProtectionData struct {
+	Protected                           *bool
+	AllowsDeletions                     *bool
+	AllowsForcePushes                   *bool
+	RequiresCodeOwnerReviews            *bool
+	RequiresLinearHistory               *bool
+	DismissesStaleReviews               *bool
+	EnforcesAdmins                      *bool
+	RequiresStatusChecks                *bool
+	RequiresUpToDateBranchBeforeMerging *bool
+	RequiredApprovingReviewCount        *int
+	// StatusCheckContexts is always available, so
+	// we don't use a pointer.
+	StatusCheckContexts []string
+	Name                string
+}
+
+// BranchProtectionsData contains the raw results
+// for the Branch-Protection check.
+type BranchProtectionsData struct {
+	Branches []BranchProtectionData
+}
+
+// RawResults contains results before a policy
```

Comment:
Nit: put the RawResults struct as the first struct in the file. Readability is better when the main struct is first and all the nested structs follow.

Reply:
makes sense. I think I inadvertently un-did what you did in the previous PR.. the RawResults was top of the file. My bad!